File: ABC/251-300/298/a.cpp
#include <bits/stdc++.h>
using namespace std;
#define rep(i,n) for(int i=0;i<(n);i++)

int main(){
    int n;
    cin >> n;
    string s;
    cin >> s;
    bool good = false;
    rep(i,n){
        if(s[i] == 'x'){
            cout << "No" << endl;
            return 0;
        }
        if(s[i] == 'o'){
            good = true;
        }
    }
    if(good) cout << "Yes" << endl;
    else cout << "No" << endl;
    return 0;
}

File: ABC/251-300/298/a.py
n = int(input())
s = input()
ans = False
for c in s:
  if c == 'o':
    ans = True
    break

for c in s:
  if c == 'x':
    ans = False
    break

if ans:
  print("Yes")
else:
  print("No")
